net/udpbroadcastrelay/src/opnsense/mvc/app/models/OPNsense/UDPBroadcastRelay/UDPBroadcastRelay.xml

```diff
@@ -50,6 +50,32 @@
             <default>0</default>
             <Required>Y</Required>
          </RevertTTL>
+         <allowcidr type="NetworkField">
+            <default></default>
+            <Required>N</Required>
+            <FieldSeparator>,</FieldSeparator>
+            <AsList>Y</AsList>
+            <NetMaskRequired>N</NetMaskRequired>
+            <NetMaskAllowed>Y</NetMaskAllowed>
+            <AddressFamily>ipv4</AddressFamily>
+            <WildcardEnabled>N</WildcardEnabled>
+         </allowcidr>
+         <blockcidr type="NetworkField">
+            <default></default>
+            <Required>N</Required>
+            <FieldSeparator>,</FieldSeparator>
+            <AsList>Y</AsList>
+            <NetMaskRequired>N</NetMaskRequired>
+            <NetMaskAllowed>Y</NetMaskAllowed>
+            <AddressFamily>ipv4</AddressFamily>
+            <WildcardEnabled>N</WildcardEnabled>
+         </blockcidr>
+         <msearch type="TextField">
+            <default></default>
+            <Required>N</Required>
+            <mask>/^(?:(?:block|fwd|proxy|dial)(?:,\S+|) )*(?:block|fwd|proxy|dial)(?:,\S+|)$/</mask>
```

Comment:
this is probably as fragile as it gets  with CSV and freestyle syntax matched via regex. :(

Reply:
Yeah... is quite the ugly one. It could just be free text with no restriction, but not sure that's any better as people will wonder why its not working 😅

I did comma separated as it needed to be split into separate args, is there a better input type to use in the UI for that?